src/Utilities/Utils.res

```diff
@@ -16,11 +16,17 @@ type dateTimeFormat = {resolvedOptions: unit => options}
 @send external remove: Dom.element => unit = "remove"
 
 @send external postMessage: (parent, JSON.t, string) => unit = "postMessage"
+@val @scope("window") external windowPostMessage: (JSON.t, string) => unit = "postMessage"
+
```

Comment:
@ArushKapoorJuspay `parent` variable is also of type `window` only. So we need not define 2 types for window and parent. 

Both window and parent variables can be of type `window`

If that is done, then the existing postMessage fn will take `window` as first arg. Then you can use that fn itself like:
```res
window->postMessage(a, b)
```